src/qml/pages/onboarding/onboarding01.qml

```diff
@@ -17,14 +18,53 @@ Page {
         anchors.fill: parent
         interactive: false
         orientation: Qt.Horizontal
-        Loader {
-            source:"onboarding01a.qml"
+        InformationPage {
+            Layout.fillWidth: true
```

Comment:
thanks for spotting these, it's a bad habit of mine to place this until the view is working correctly. Since you've already opened a PR to address them, i've left this as is.